src/utils/format-srcset/index.js

```diff
@@ -0,0 +1,14 @@
+const formatSrcset = (src, resizerOptions, width, height) =>
```

Comment:
can this also be used in the image component? I know that would be a helpful work for reusability that we were talking about

Reply:
@JackHowa : Yes, this util was made to format the Resizer v2 params and dimensional options for the `Image` querystring as per these lines:
https://github.com/WPMedia/arc-themes-components/blob/arc-themes-release-version-2.0.1/src/components/image/index.jsx#L36-L48

I wasn't certain if changing `Image` to use this utility was scoped in this ticket. That being said, I feel that it's a small-effort, high-reward change and I'd be happy to include it. Please let me know how you'd like me to proceed. Thanks!

Reply:
yeah good idea! I think adding the util to the image was something we talked about and it's in the ticket: 

> The interface for configuring the <Image> component will match the <Picture.Source>: that is deliberate. They should share logic in some instances in the util folder. 

<img width="768" alt="Screen Shot 2022-08-01 at 17 33 53" src="https://user-images.githubusercontent.com/5950956/182256771-e75447e2-5ec7-4f64-bef4-5816ab34274d.png">

Reply:
@JackHowa : Thank you. I have integrated the utility with image. The changes are ready for review.

Reply:
I addressed.